inc/return.h

```diff
@@ -60,7 +60,7 @@
 	FuncObj->byteswapped = 1;				\
```

Comment:
This code fragment also shows up in tosfns.h, SWAPPED_FN_CHECK expanding to this if RESWAPPEDCODESTREAM, or nothing if not.  Not sure if it's worth shrinking the FastRetCALL down here.

Reply:
If this code is made to work again ... I suspect we'd be able to simplify a bunch of it. But even when I did this, things didn't compile afterwards and then I got sick for a day and I forgot where I was at with it.